src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1762,6 +1762,33 @@ public void recallNoticeToLoanOwnerIsNotSendWhenDueDateIsNotChanged()
       sentNotices, Matchers.empty());
   }
 
+  @Test
+  public void canCreatePagedRequestWithNullProxyUser()
+    throws InterruptedException,
+    ExecutionException,
+    TimeoutException,
+    MalformedURLException {
+
+    //Set up the item's initial status to be AVAILABLE
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final String itemInitialStatus = smallAngryPlanet.getResponse().getJson().getJsonObject("status").getString("name");
+    assertThat(itemInitialStatus, is(ItemStatus.AVAILABLE.getValue()));
+
+    //Attempt to create a page request on it.  Final expected status is PAGED
+    final IndividualResource servicePoint = servicePointsFixture.cd1();
+    final IndividualResource pagedRequest = requestsClient.create(new RequestBuilder()
+      .page()
+      .forItem(smallAngryPlanet)
+      .withPickupServicePointId(servicePoint.getId())
+      .by(usersFixture.james())
+      .withUserProxyId(null));
```

Comment:
I think it is likely the builder will just omit this and so the property won’t be included (which the default builder should provide).

It is difficult to verify automatically that the HTTP request to users won’t occur. I’ll attempt to mitigate that shortcoming of API style testing with a follow up that changes the fakes to be stricter.